apps/web/app/[locale]/auth/passcode/page.tsx

```diff
@@ -101,10 +101,13 @@ function EmailScreen({ form, className }: { form: TAuthenticationPasscode } & IC
 
 					<div className="flex items-center justify-between w-full mt-6">
 						{/* Send code */}
-						<div className="flex flex-col items-start">
-							<Link href="/auth/team">
-								<BackButton />
-							</Link>
+						<div className="flex flex-col items-start gap-2">
+							<div className="flex justify-start items-center text-sm">
+								<span>{t('common.BACK')}</span>
+								<Link href="/auth/team" className="text-primary">
+									<span>Register now</span>
```

Comment:
Why "Register now" is not in the translation files? Also, again small detail - it should be "Register Now!".